content/en/blog/_posts/2021-08-09-memory-manager-moves-to-beta.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: blog
+title: "Kubernetes Memory Manager moves to beta"
+date: 2021-08-09
+slug: kubernetes-1-22-feature-memory-manager-moves-to-beta
+---
+
+**Authors:** Artyom Lukianov (Red Hat), Cezary Zukowski (Samsung)
+
+# Memory manager moves to beta
```

Comment:
Omit this heading - the Hugo theme we use adds a heading at `<h1>` level automatically, based on the page title.

Reply:
done